library/alloc/src/boxed.rs

```diff
@@ -343,6 +343,29 @@ impl<T, A: Allocator> Box<T, A> {
         let (raw, alloc) = Box::into_raw_with_allocator(boxed);
         unsafe { Box::from_raw_in(raw as *mut [T; 1], alloc) }
     }
+
+    /// Consumes the `Box`, returning the wrapped value.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(box_into_inner)]
+    ///
+    /// let c = Box::new(5);
+    ///
+    /// let five = c.into_inner();
+    /// ```
+    #[unstable(feature = "box_into_inner", issue = "80437")]
+    #[inline]
+    pub fn into_inner(self) -> T {
```

Comment:
```suggestion
    pub fn into_inner(b: Self) -> T {
```

This is `Deref` type, so it shouldn't have methods that could shadow methods of inner structure.

Reply:
Thanks for pointing out. Updated.

Reply:
Ah, no wonder I was  wondering why `into_raw` was so weird since it was taking in `Box<T>`, now that clears my confusion.